includes/bootstrap.inc

```diff
@@ -273,7 +274,7 @@ function drush_bootstrap_validate($phase) {
         break;
       }
       if ($phase_index > $validated_phase) {
-        $current_phase .= '_validate';
+        $current_phase .= 'Validate';
```

Comment:
Oh, I misread. I thought the drush invoke function names were changing. I've merged this PR.